_posts/2019-12-09-ebcdic-character-format-guide.markdown

```diff
@@ -97,28 +80,40 @@ Let's start off showing a fictional fixed width copybook.
           10  FILLER               PIC X(50).
 ```
 
-WHAT DOES PIC MEAN?
+The spaces are important as that is a different section that is reserved for things such as comments.
```

Comment:
This sentence doesn't really make a lot of sense - consider revising. Indentation is significant, what space is used for comments etc?